Documentation: Clarify instructions about changing row_factory for SQLite3

SilverWingedSeraph opened the pull request:
When I was working through the tutorial, this was very confusing to me; so, I've added the code and clarification that would have helped me get through it faster.

jeffwidman commented:
@SilverWingedSeraph Are you planning on updating this?

SilverWingedSeraph commented:
I totally forgot about this, sorry! I'll fix it now.

jeffwidman commented on docs/patterns/sqlite3.rst:
```diff
@@ -71,18 +71,36 @@ Now in each request handling function you can access `g.db` to get the
 
     db.row_factory = sqlite3.Row
 
+This would use Row objects rather than dicts to return the results of queries. These are `namedtuple` s, so we can access them either by index or by key. For example, assuming we have a ``sqlite3.Row`` called ``r`` for the rows id, FirstName, LastName, and MiddleInitial::
```
I think `namedtuples` needs two `` since it's restructured text and not markdown

SilverWingedSeraph commented:
I'm sorry I've been late on replying. I just graduated from high school and should be much more responsive in the future.

I fixed the formatting issue and changed the code to demonstrate some of the more useful capabilities of the Row object.

jeffwidman commented:
Thanks!
